src/Models/Category.php

```diff
@@ -95,14 +98,18 @@ public function rewrites(): HasMany
             ->where('entity_type', 'category');
     }
 
-    public function getParentcategoriesAttribute()
+    public function parentcategories(): Attribute
     {
-        $categoryIds = explode('/', $this->path);
-        $categoryIds = array_slice($categoryIds, array_search(config('rapidez.root_category_id'), $categoryIds) + 1);
-
-        return ! $categoryIds ? collect() : Category::whereIn($this->getQualifiedKeyName(), $categoryIds)
-            ->orderByRaw('FIELD(' . $this->getQualifiedKeyName() . ',' . implode(',', $categoryIds) . ')')
-            ->get();
+        return Attribute::make(
```

Comment:
We'll probably want to add shouldCache here
https://laravel.com/docs/master/eloquent-mutators#accessor-caching